drivers/adc/ad713x/ad713x.c

```diff
@@ -318,6 +318,24 @@ int32_t ad713x_mag_phase_clk_delay_chan(struct ad713x_dev *dev,
  * 							 CH3
  * @return 0 in case of success, -1 otherwise.
  */
+
+ int32_t ad713x_channel_sync(struct ad713x_dev *dev)
```

Comment:
ad713x.c and ad713x.h contain the multisync feature, so bundle only these 2 in one commit.

The rest of the changes are separate things (like the SPI mode and ref clk etc.) and need to be presented as separate commits and documented in the commit description with a few sentences.